pytorch_lightning/utilities/enums.py

```diff
@@ -68,6 +68,12 @@ def __call__(cls, *args: Any, **kwargs: Any) -> Any:
         return obj
 
 
+class DeprecatedEnum(LightningEnum, metaclass=DeprecatedEnumMeta):
```

Comment:
```suggestion
class DeprecatedEnum(LightningEnum, metaclass=DeprecatedEnumMeta):
```
let's keep this protected?

Reply:
I don't see a change -- what is your concern and suggestion?

Reply:
updated!

Reply:
I also added an `_` to `DeprecatedEnumMeta`.